actions/src/main/java/com/redhat/service/bridge/actions/webhook/WebhookAction.java

```diff
@@ -45,9 +48,15 @@ public WebhookActionValidator getParameterValidator() {
     }
 
     @Override
-    public ActionInvoker getActionInvoker(ProcessorDTO processor, BaseAction baseAction) {
+    public ActionInvoker getActionInvoker(ProcessorDTO processor, BaseAction baseAction, GlobalConfig globalConfig) {
```

Comment:
@r00ta Seeing as this `GlobalConfig` only applies to the WebHook Action, can't we inject it into this class as a bean and then pass it as a constructor param to the `WebHookInvoker`.

That would avoid us updating the method signatures for all our interfaces.

Reply:
@robpblake that would work if we refactor the `actions` module to properly separate the actions definition and the implementations. The `GlobalConfig` bean is supposed to be produced by the `executor` application, and the `actions` module is being used also by the `manager` that does not provide such `bean`. 
The design problem I see here is that the manager should not include the implementation of the actions, because for example since day 0 the manager is bringing in the kafka client from the `sendToKafka` action..

Reply:
If we move away from `GlobalConfig` and we inject directly only the optional property it would work I think. Let me try

Reply:
Done